apps/server/lib/lexical/server/code_intelligence/completion/translations/callable.ex

```diff
@@ -82,4 +87,12 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.Callable do
   defp name_and_arity(%_{name: name, arity: arity}) do
     "#{name}/#{arity}"
   end
+
+  defp reduce_args_when_pipe(%{argument_names: argument_names} = callable, %Env{} = env) do
```

Comment:
I don't like modifying the original callable and sending it through. I think it's a lot clearer to have a function that operates on the argument names;  I don't think this is an improvement in clarity over the original.

Reply:
the original had a bug though, if it was a zero arg function, it would fail.

Reply:
I created a function named normalize_argument_names, and `callable_snippet/2` and `label/2` both require context awareness, so passing "env" to them makes sense.

Reply:
that's much licer.